### PR TITLE
fix(VCombobox): do not add search to list when selecting items with keyboard

### LINKED ISSUE #8841: [Bug Report] Combobox add 2 items when pressing enter on a search item
### Environment
**Vuetify Version:** 2.0.13
**Vue Version:** 2.6.10
**Browsers:** Chrome 76.0.3809.132
**OS:** Android 9, Windows

### Steps to reproduce
- in the tags field, type `g`
- press the `arrow down` to select the first option `Gaming`
- press `enter`

### Expected Behavior
1 chip is added: `Gaming`

### Actual Behavior
2 chips are added: `Gaming` and `g`

### Reproduction Link
<a href="https://codepen.io/gotson/pen/GRKOoeb" target="_blank">https://codepen.io/gotson/pen/GRKOoeb</a>

### PR DESCRIPTION
**Pull request by haggys22**
## Description
There was a timing issue with `this.$nextTick`, triggering the selection twice: once for the selected `VMenu` item and once for the search term when the menu index is `-1`.

See https://github.com/vuetifyjs/vuetify/issues/8841#issuecomment-562203859 for the code execution order without this fix.

## Motivation and Context
This fixes #8841. Also the workaround for #8842 works again.

## How Has This Been Tested?
visually

## Markup
<details>

```vue
<template>
  <v-container>
    <v-combobox
            v-model="model"
            :items="items"
            :search-input.sync="search"
            hide-selected
            label="Add some tags"
            multiple
            persistent-hint
            chips
    ></v-combobox>
  </v-container>
</template>

<script>
export default {
  data: () => ({
    items: ['Gaming', 'Programming', 'Vue', 'Vuetify'],
    model: [],
    search: null,
  }),
  watch: {
    model() {
      this.search = null
    }
  }
}
</script>
```
</details>

## Types of changes

- [x] Bugfix (non-breaking change which fixes a bug)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by haggys22** [comment]
For some reason the test

```
FAIL Vuetify src/components/VSelect/__tests__/VSelect4.spec.ts
  ● VSelect.ts › should emit listIndex event when navigated by keyboard

    expect(jest.fn()).toHaveBeenCalledWith(expected)

    Expected mock function to have been called with:
      0
    as argument 1, but it was called with
      -1.
```

fails, but inspecting a VSelect with Vue devtools shows that `update:list-index` is emitted with the correct index when navigating with the keyboard

**Comment by haggys22** [comment]
@johnleider 
* Updated branch
* Added `Playground.vue` to PR message
* Fixed failing test

**Comment by johnleider** [comment]
Testing your branch and the issue still persists.

select

File: packages/vuetify/src/components/VCombobox/VCombobox.ts [hunk 1]
```diff
@@ -100,10 +100,10 @@ export default VAutocomplete.extend({
     // to overwrite removal in v-autocomplete
     onEnterDown (e: Event) {
       e.preventDefault()
+      // If has menu index, let v-select-list handle
+      if (this.getMenuIndex() > -1) return
 
       this.$nextTick(() => {
```
Comment (jacekkarczmarczyk) [inline packages/vuetify/src/components/VCombobox/VCombobox.ts]: This could be `this.$nextTick(this.updateSelf)`

**Comment by haggys22** [comment]
With my latest changes the v1.5.18 behaviour is restored, i.e. after pressing `enter` only the selected item is added as a chip and the search term remains in the input field

**Comment by johnleider** [comment]
Determined that #8841 was a duplicate of #6697

**Pull request merged**